cicd/jenkins/openshift-jenkins.adoc

```diff
@@ -28,4 +28,6 @@ include::modules/customizing-the-jenkins-image-stream-tag.adoc[leveloffset=+1]
 * link:https://catalog.redhat.com/software/containers/search?q=Jenkins%202&p=1[Certified `jenkins` images]
 * link:https://catalog.redhat.com/software/containers/search?q=Jenkins%20Agent%20Base&p=1[Certified `jenkins-agent-base` images]
 * link:https://catalog.redhat.com/software/containers/search?q=jenkins-agent-maven&p=1[Certified `jenkins-agent-maven` images]
+// KEEP?
 * link:https://catalog.redhat.com/software/containers/search?q=jenkins-agent-nodejs&p=1[Certified `jenkins-agent-nodejs` images]
+// KEEP?
```

Comment:
Same here, we can keep it, even it is not the favorite way of using agent and node agents.